depends/packages.md

```diff
@@ -152,8 +152,7 @@ Most autotools projects can be properly staged using:
 
 ## Build outputs:
 
-In general, the output of a depends package should not contain any libtool
```

Comment:
This line can remain: https://github.com/bitcoin/bitcoin/pull/30902#discussion_r1759727845.

Reply:
no problem removed in [03fd466](https://github.com/bitcoin/bitcoin/pull/30875/commits/03fd466581666cfd637e79a8594a9a0e7d097524)[03fd466](https://github.com/bitcoin/bitcoin/pull/30875/commits/03fd466581666cfd637e79a8594a9a0e7d097524)[03fd466](https://github.com/bitcoin/bitcoin/pull/30875/commits/03fd466581666cfd637e79a8594a9a0e7d097524)